templates/preview-a4a.html

```diff
@@ -5,15 +5,13 @@
     <meta name="viewport" content="width=device-width,minimum-scale=1">
     {{> head.html}}
     <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
-    {{#metadata.adSlot}}
+     {{#metadata.adSlot}}
       <script async src="https://cdn.ampproject.org/v0.js"></script>
       <script async custom-element="amp-ad" src="https://cdn.ampproject.org/v0/amp-ad-0.1.js"></script>
     {{/metadata.adSlot}}
     {{^metadata.adSlot}}
-      <!-- TODO(aghassemi,#9311): Remove when A4A can be embeded using prod binaries -->
-      <script async src="https://amp-ads.firebaseapp.com/dist/amp.js"></script>
-      <script async custom-element="amp-ad" src="https://amp-ads.firebaseapp.com/dist/v0/amp-ad-0.1.max.js"></script>
-      <script async custom-element="amp-ad-fake" src="https://amp-ads.firebaseapp.com/dist/v0/amp-ad-network-fake-impl-0.1.max.js"></script>
+      <script async src="https://cdn.ampproject.org/v0.js"></script>
+      <script async custom-element="amp-ad" src="https://cdn.ampproject.org/v0/amp-ad-0.1.js"></script>
```

Comment:
Do we then still need the adSlot variable? Seems like both branches now do the same.

Reply:
I believe not. I'll remove adSlot

Reply:
I thought about this. Actually we would still need this. Because when we compare the performance between AMP Ads and HTML Ads, we don't want to use fake ad to cheat. We would still need the adSlot variable to request a real doubleclick ad to compare.

Reply:
Ahhh I see your point. Removed